docs/adr/001_encoder.md

```diff
@@ -40,8 +40,8 @@ We will use the `MJPEGEncoder` that is using the Raspberry Pi's hardware.
 
 ## Consequences
 
-* Following the documentation this encoder will consume less CPU than the software encoder
-* Too many simultaneous streams will put more load on the CPU than the software encoder.
+* Following the documentation and some experiments this encoder will consume less CPU than the software encoder.
+* This encoder is only available on Raspberry Pi hardware
```

Comment:
Are you sure about that one? As it inherits from v4l2_encoder.py it should use v4l2.

Reply:
This is how I interpret the documentation:
> The MJPEGEncoder class implements an MJPEG encoder using the Raspberry Pi’s in-built hardware, accessed through the V4L2 kernel drivers.

Reply:
I have understood it in such a way that the Raspberry Pi’s in-built hardware is used because they access it through the V4L2 kernel drivers that would access the given hardware on other devices too if they support the V4L2 encoding.

Looking at the [code](https://github.com/raspberrypi/picamera2/blob/main/picamera2/encoders/v4l2_encoder.py#L35) you are right. It sends everything to `/dev/video11` and this seems to bee the [encoder](https://forums.raspberrypi.com/viewtopic.php?f=43&t=62364&p=1527503#p1527503) of V4L2.
As long as other devices don't have it on `/dev/video11` it won't work.